examples/complete/main.tf

```diff
@@ -64,6 +64,16 @@ module "opensearch" {
     dedicated_master_type    = "c6g.large.search"
     instance_type            = "r6g.large.search"
 
+    node_options = {
```

Comment:
I don't think this is quite right, did we try deploying this? 

I think its:

```hcl
    node_options = {
      node_type = "coordinator"
      node_config = {
        enabled = true
        count   = 3
        type    = "m6g.large.search"
      }
    }
```

Reply:
I was able to deploy it successfully. It looked like `node_options` is a list that could pass more than one set of options in the future (right now there's only `coordinator`) so I made it accept [either the key or a value in the list](https://github.com/terraform-aws-modules/terraform-aws-opensearch/pull/37/files#diff-dc46acf24afd63ef8c556b77c126ccc6e578bc87e3aa09a931f33d9bf2532fbbR123) called `node_type`. But I can change it if that seems unlikely.

Reply:
ok, lets try this